examples/vm_scheduling/rule_based_algorithm/config.yml

```diff
@@ -30,3 +30,4 @@ algorithm:
   # args:
     # pm_num: 100
```

Comment:
for pm_num and pm_cpu_core_num, get it from env would be better, especially the configuration of PMs can be various